pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -1875,6 +1884,29 @@ class FixProcessor {
     }
   }
 
+  /// This fix is not going to work well if what you want is to remove only a
+  /// TypeName inside a FunctionTypedFormalParameter (returnType, parameter or
+  /// generic) so if you want to do that you should create other fix or modify
+  /// this one carefully.
```

Comment:
I didn't see your other comment. I'll respond to it, and you can ignore this comment.